configs/skymap_small.py

```diff
@@ -1,9 +1,10 @@
 config.skyMap["discrete"].projection = "TAN"
-# center of tract 3828 patch 3,3=24
-config.skyMap["discrete"].raList = [56.64971166122064]
-config.skyMap["discrete"].decList = [-36.4462532139527947602893]
-# 28000 (pix as in DC2 skymap; 7x4000pix patches)/3600(deg/asec)*0.2(asec/pix)/2(diam-to-rad) = 7/9
-# Then round down to 6.4/9, otherwise it seems to make 8x8 patches
+# this should be the center of tract 3828 patch 3,3=24
+# ... but that didn't work, so it's those coords (see skymap.py) plus a mysterious diff
+config.skyMap["discrete"].raList = [57.34525755993447]
+config.skyMap["discrete"].decList = [-35.89271688916593]
+# 28000 (pix as in DC2 skymap; 14x2000pix patches)/3600(deg/asec)*0.2(asec/pix)/2(diam-to-rad) = 7/9
+# Then round down to 6.4/9, otherwise it seems to make 8x8 4k patches (this works for 2k too)
```

Comment:
I realize you haven't changed this comment, but I don't understand "round down to 6.4/9". And can you say what this _is_ supposed to make, if you don't want it to make 8x8 patches?

Reply:
The 4k x 4k skymap is supposed to make 7x7 patches, just like the standard DC2 skymap. So I thought I did the math correctly but evidently putting exactly 7/9 made the tracts with 8x8 patches, larger than I wanted. I don't really understand why one has to specify a tract radius when they're (nearly?) rectangular in ra/dec, so it's fair to say that I just made an educated guess at reducing the radius until it ended up with 7x7 patches.

For the 2kx2k skymap, I intended to make 13x13 patches so that the central patch covers the central quarter of the area of the central patch in the 4kx4k skymap, and it happened to work out like this. I'll update the comments to be more clear about this. There is a side effect that the tract is slightly smaller (being 13x13 instead of 14x14) but I think that's fine, there's no intent to process anything all the way out to the tract's edge.

... but thanks for pointing this out, as it made me realize that I hadn't changed the patch number in the data query in the run script when using the small 2kx2k skymap.

Reply:
Note that I updated this again - it turns out the reason I had to offset the tract center in the small skymap was precisely because I had picked the wrong patch. Everything pretty much works as intended now and I've expanded on the comments to clarify.